netjsonconfig/backends/airos/converters.py

```diff
@@ -0,0 +1,679 @@
+
+    def host(self):
+        original = get_copy(self.netjson, 'general', {})
+        return {
+            'host': [{
+                'name': original.get('hostname', 'airos'),
+                'status': 'enabled',
+            }],
+        }
+
+    def nameserver(self):
+        result = []
+        original = get_copy(self.netjson, self.netjson_key, [])
+        for nameserver in original:
+            result.append({
+                'ip': nameserver,
+                'status': 'enabled',
+            })
+        return {'nameserver': result}
+
+    def to_intermediate(self):
+        result = []
+        result.append(self.host())
+        result.append(self.nameserver())
+        result.append({'status': 'enabled'})
+        return (('resolv', result),)
+
+
+class Route(AirOsConverter):
+    netjson_key = 'routes'
+
+    def to_intermediate(self):
+        result = []
+        original = get_copy(self.netjson, self.netjson_key, [])
+        routes = []
+        for r in original:
+            network = ip_interface(r['destination'])
+            temp = {}
+            temp['ip'] = str(network.ip)
+            temp['netmask'] = str(network.netmask)
+            routes.append({
+                'gateway': r['next'],
+                'ip': temp['ip'],
+                'netmask': temp['netmask'],
+                'status': 'enabled',
+            })
+        result.append(routes)
+        result.append({'status': 'enabled'})
+        return (('route', result),)
+
+
+class Snmp(AirOsConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = [
+            {
+                'community': 'public',
+                'contact': '',
+                'location': '',
+                'status': 'enabled',
+            },
+        ]
+        return (('snmp', result),)
+
+
+class Sshd(AirOsConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+        result.append({
+            'auth': {'passwd': 'enabled'},
+            'port': 22,
+            'status': 'enabled',
+        })
+        return (('sshd', result),)
+
+
+class Syslog(AirOsConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+
+        result.append({
+            'remote': {
+                'port': 514,
+                'status': 'disabled',
+            },
+            'status': 'enabled',
+        })
+        return (('syslog', result),)
+
+
+class System(AirOsConverter):
+    netjson_key = 'general'
+
+    def to_intermediate(self):
+        result = []
+        result.append({
+            'airosx': {
+                'prov': {
+                    'status': 'enabled',
+                },
+            },
+            'cfg': {
+                'version': 0,
+            },
+            'date': {
+                'status': 'enabled',
+            },
+            'external': {
+                'reset': 'enabled',
+            },
+            'timezone': 'GMT',
```

Comment:
we'll need to implement this soon, it shouldn't have priority right now but we will have to do it in the 3rd phase